docs/development.rst

```diff
@@ -0,0 +1,119 @@
+Development
+===========
+
+Requirements
+------------
+
+* `python`_
+
+.. _python: https://www.python.org/
+
+
+Install Locally
+---------------
+
+#. `Clone`_ and change to the directory::
+
+    git clone git@github.com:groovecoder/push-dev-dashboard.git
+    cd push-dev-dashboard
+
+#. Create and activate a `virtual environment`_ (Can also use `virtualenvwrapper`_)::
+
+    virtualenv env
+    source env/bin/activate
+
+#. `Install requirements`_::
+
+    pip install -r requirements.txt
+
+#. Source the ``.env`` file to set environment config vars (Can also use `autoenv`_)::
+
+    source .env
+
+#. `Migrate`_ DB tables ::
+
+    python manage.py migrate
+
+#. `Create a superuser`_::
+
+    python manage.py createsuperuser
+
+.. _Clone: http://git-scm.com/book/en/Git-Basics-Getting-a-Git-Repository#Cloning-an-Existing-Repository
+.. _Install requirements: http://pip.readthedocs.org/en/latest/user_guide.html#requirements-files
+.. _Create a superuser: https://docs.djangoproject.com/en/1.7/ref/django-admin/#django-admin-createsuperuser
+
+
+Run it
+------
+
+#. Source the ``.env`` file to set environment config vars (Can also use `autoenv`_)::
+
+    source .env
```

Comment:
I notice this is also done earlier in the steps (line 31 currently). Does it need to be done again?